test/Test/Toml/PrefixTree.hs

```diff
@@ -0,0 +1,13 @@
+module Test.Toml.PrefixTree
```

Comment:
I really like this structure! :+1:  It makes code easier to read with this test separation